GETTING_STARTED.md

```diff
@@ -46,7 +46,7 @@ RSpec.configure do |config|
 end
 ```
 
-Remember to require the above file in your rails_helper since the support folder isn't eagerly loaded
+Remember to require the below file in your rails_helper since the support folder isn't eagerly loaded
```

Comment:
I see what you mean here, but we do want `above` in this context. I think what we're trying to say here is something like:

> Remember to `require` the above file since the `support` folder isn't automatically loaded for specs. You can do so by adding the following line in your `rails_helper.rb`:

Does that clarify the intent here?